modules/luci-base/htdocs/luci-static/resources/ui.js

```diff
@@ -2265,8 +2266,106 @@ var UIDynamicList = UIElement.extend(/** @lends LuCI.ui.DynamicList.prototype */
 			this.addItem(dl, this.values[i], label);
 		}
 
+		this.initDragAndDrop(dl);
+
 		return this.bind(dl);
 	},
+	initDragAndDrop: function(dl) {
+		let draggedItem = null;
+		let touchStartY = 0;
```

Comment:
drop this. it's unused.

Reply:
i guess you mean the touchStartY. yes that was a leftover from a test.